directory/src/main/java/org/ecclesia/directory/domain/Group.java

```diff
@@ -1,8 +1,10 @@
 package org.ecclesia.directory.domain;
 
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
-@Data
+@Getter @Setter @NoArgsConstructor
```

Comment:
**Reporter**: Checkstyle
**Rule**: AnnotationLocationMostCases
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-6/directory/src/main/java/org/ecclesia/directory/domain/Group.java L7

Annotation &#39;Setter&#39; should be alone on line.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a679695898>*